src/com/palmergames/bukkit/util/BukkitTools.java

```diff
@@ -285,4 +287,17 @@ public static Objective objective(Scoreboard board, @NotNull String name, @NotNu
 		}
 		return objective;
 	}
+
+	/**
+	 * @param event The event to call
+	 * @return {@code false} if the event is cancellable and was cancelled, otherwise {@code true}.
+	 */
+	public static boolean callEvent(@NotNull Event event) {

```

Comment:
This might be more readable/make more sense if it had a name similar to ifEventIsUncancelled() or ifEventFiredSuccessful()